src/goto-programs/interpreter.cpp

```diff
@@ -960,7 +960,8 @@ bool interpretert::unbounded_size(const typet &type)
 size_t interpretert::get_size(const typet &type)
 {
   if(unbounded_size(type))
-    return 2ULL << 32ULL;
+    return 1ULL << 31ULL;
+    //return 2ULL << 32ULL;
```

Comment:
this is required for 32 bit systems, as the original code is UB for those, it's not yet obvious how to fix this in general for 32 bit
Still, it should not be part of this PR